packages/esm-service-queues-app/src/metrics/metrics-cards/metrics-card.scss

```diff
@@ -23,11 +23,30 @@
 .totalsLabel {
   @include type.type-style('label-01');
   color: $text-02;
+  margin-bottom: layout.$spacing-02;
 }
 
 .totalsValue {
   @include type.type-style('heading-04');
   color: $ui-05;
+  margin: 0;
+}
+
+.metricsContainer {
+  display: flex;
+  gap: layout.$spacing-05;
+  align-items: flex-end;
+}
+
+.metricItem {
+  display: flex;
+  flex-direction: column;
+
+  &:nth-child(2) {
```

Comment:
Using `nth-child` here feels fragile to me. Wouldn't it be better to use a semantic class name instead? Something like: 


```tsx
 {showUrgent && (
    <div className={styles.metricItem}>
      <label className={styles.urgentLabel}>{t('urgent', 'Urgent')}</label>
      <p className={styles.urgentValue}>{urgentCount ?? '0'}</p>
    </div>
  )}
```

```scss
.urgentValue {
  @include type.type-style('heading-04');
  color: red;
}
```